.travis.yml

```diff
@@ -23,4 +23,4 @@ script:
 after_success:
   - codecov
 before_script:
-  - mongo admin --eval 'db.createUser({user:"travis",pwd:"test",roles:[{role:"root",db:"admin"}]});'
```

Comment:
Purpose of this change? This is only used for the Travis CI test harness

Reply:
This was changed to be consistent with hardcoded uri values used when initializing mongo db. It is also consistent with the default credentials used by the mongo db docker image.